cluster/gce/coreos/master.yaml

```diff
@@ -135,6 +135,7 @@ coreos:
         --reconcile-cidr=false
         Restart=always
         RestartSec=10
+        KillMode=process
```

Comment:
The side effect is that we probably leak master spawned (if any) process when systemd restarts master.